ibis/backends/pandas/execution/window.py

```diff
@@ -210,16 +216,20 @@ def trim_window_series(data, timecontext: Optional[TimeContext]):
     if time_col not in df:
         return data
 
-    # if Series dosen't contain a name, reset_index will assign
-    # '0' as the column name for the column of value
-    name = data.name if data.name else 0
     subset = df.loc[df[time_col].between(*timecontext)]
 
-    # get index columns for the Series
-    non_target_columns = list(subset.columns.difference([name]))
+    # Get columns to set for index
+    if isinstance(data, pd.Series):
+        # if Series dosen't contain a name, reset_index will assign
+        # '0' as the column name for the column of value
+        name = data.name if data.name else 0
+        index_columns = list(subset.columns.difference([name]))
+    else:
+        name = data.columns.tolist()
```

Comment:
you don't need the `.tolist()` here just pass name directly to `.difference`

Reply:
Oops forgot to remove this, +1